classes/admin/class-kp-form-fields.php

```diff
@@ -134,6 +134,11 @@ class KP_Form_Fields {
 			'currency' => 'SEK',
 			'endpoint' => '',
 		),
+		'sk' => array(
```

Comment:
This should be above Spain right for it to follow the alphabetical order of the country names?